project/cmake/CMakeLists.txt

```diff
@@ -24,7 +24,6 @@ option(ENABLE_PLIST       "Enable AirPlay support?" ON)
 option(ENABLE_NFS         "Enable NFS support?" ON)
 option(ENABLE_AIRTUNES    "Enable AirTunes support?" ON)
 option(ENABLE_CEC         "Enable CEC support?" ON)
-option(ENABLE_OPENSSL     "Enable OpenSSL?" ON)
```

Comment:
Thanks. This option tells CMake to find the OpenSSL header and lib and link it to the Kodi executable. Curl and other dependencies on Windows are still being downloaded in binary form, so if that has OpenSSL all should be good.